src/seq/index.rs

```diff
@@ -285,7 +303,15 @@ where
     X: Into<f64>,
 {
     if length > (u32::MAX as usize) {
-        sample_efraimidis_spirakis(rng, length, weight, amount)
+        #[cfg(target_pointer_width = "32")]
+        unreachable!();
+
+        #[cfg(target_pointer_width = "64")]
+        {
```

Comment:
I think these `cfg`s can be skipped, since this branch will be eliminated either way.

Reply:
Done. I left one `cfg` there just to make it obvious that this code is for 64-bit platforms only.

Reply:
Removing: it doesn't compile. `i686` needs valid code and the `u64` impl is not available.